### PR TITLE
Section 1 completed

### PR DESCRIPTION
**Pull request by sriram-500apps**
In this section I learned about JSON and difference between array and JSON.
I also learned about JSON parse and stringify methods.

### CODE REVIEW

File: section01/Index2.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    var myObj = { "people": [{ "firstName": "Laurence", "lastName": "Smith", "age": 50 }, { "firstName": "John", "lastName": "Jone", "age": 50 }] };
+    myObj.people[1].monthsknown = "18";
+
+    //sessionStorage.setItem('friend',JSON.stringify(myObj) );
+
+    var myList = JSON.parse(sessionStorage.getItem('friend'));
+    console.log(myList);
+
+
+    var strpeople = JSON.stringify(myObj);
```
Comment (Sriram48) [inline section01/Index2.html:13]: comment what stringify do.

File: section01/Index2.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    /// console.log(strpeople );
+
+
+    var myfriends = JSON.parse(strpeople);
```
Comment (Sriram48) [inline section01/Index2.html:17]: comment what parse do.

File: section01/Index2.html (hunk 1 above)
Comment (sriram-500apps) [reply section01/Index2.html:13]: done

File: section01/Index2.html (hunk 2 above)
Comment (sriram-500apps) [reply section01/Index2.html:17]: done

**Pull request merged**